### ISSUE TITLE
UNMET PEER DEPENDENCY react@16.0.0

### ISSUE THREAD

**Issue by studioraygun** [reactjs/react-chartjs#194]
Why am I getting this when trying to install react-chartjs on a fresh install of create-react-app?

package.json looks like this:

```{
  "name": "charts",

  "version": "0.1.0",

  "private": true,

  "devDependencies": {

    "react-scripts": "1.0.14"

  },

  "dependencies": {

    "react": "^16.0.0",

    "react-chartjs": "^0.8.0",

    "react-dom": "^16.0.0"

  },

  "scripts": {

    "start": "react-scripts start",

    "build": "react-scripts build",

    "test": "react-scripts test --env=jsdom",

    "eject": "react-scripts eject"

  }
}
```

**Comment by cardacci** [comment]
I think that this library does not support the new update 16 of React, you can see my thread about it, it might be useful: [https://github.com/reactjs/react-chartjs/issues/195](https://github.com/reactjs/react-chartjs/issues/195)

**Comment by Jareechang** [comment]
I believe this has been fixed by #192. Closing issue.
